.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Help & Questions
```

The Help & Question page already uses this emoji, let's keep it unified:

```suggestion
  - name: 💬 Help & Questions
```